LiteNetLibSystemExample/core/GameManager.cs

```diff
@@ -9,12 +10,12 @@ public enum SupportedOS
     Haiku = 1 << 1,
     iOS = 1 << 2,
     HTML5 = 1 << 3,
-    OSX = 1 << 4,
+    macOS = 1 << 4,
     Server = 1 << 5,
     Windows = 1 << 6,
     UWP = 1 << 7,
     X11 = 1 << 8,
```

Comment:
In Godot 4.4, it changes from X11 to Linux.

```suggestion
    Linux = 1 << 8,
```